sp041_20191205T1757-instrument-scalar_v2.cdl

```diff
@@ -0,0 +1,417 @@
+                CHLA:sensor = "INSTRUMENT_FLUOROMETER_CHLA";
+	byte CHLA_QC(N_MEASUREMENTS) ;
+		CHLA_QC:_FillValue = 0b ;
+		CHLA_QC:long_name = "quality flag" ;
+	float DOXY(N_MEASUREMENTS) ;
+		DOXY:_FillValue = NaN ;
+		DOXY:long_name = "Dissolved oxygen" ;
+                DOXY:vocabulary = "http://vocab.nerc.ac.uk/collection/OG1/current/DOXY/";
+		DOXY:standard_name = "moles_of_oxygen_per_unit_mass_in_sea_water" ;
+		DOXY:units = "micromol kg-1" ;
+		DOXY:ancillary_variables = "DOXY_QC" ;
+		DOXY:valid_max = 0. ;
+		DOXY:valid_min = 500. ;
+		DOXY:coverage_content_type = "physicalMeasurement" ;
+		DOXY:comment = "Concentration of dissolved oxygen per unit mass of the water column. Oxygen may be expressed in terms of mass, volume or quantity of substance" ;
+                DOXY:sensor = "INSTRUMENT_OPTODE_DOXY";
+	byte DOXY_QC(N_MEASUREMENTS) ;
+		DOXY_QC:_FillValue = 0b ;
+		DOXY_QC:long_name = "quality flag" ;
+	double DEPLOYMENT_TIME ;
+		DEPLOYMENT_TIME:long_name = "date of deployment" ;
+		DEPLOYMENT_TIME:standard_name = "time" ;
+		DEPLOYMENT_TIME:calendar = "gregorian" ;
+		DEPLOYMENT_TIME:units = "seconds since 1970-01-01T00:00:00Z" ;
+		DEPLOYMENT_TIME:axis = "T" ;
+	double DEPLOYMENT_LATITUDE ;
+		DEPLOYMENT_LATITUDE:long_name = "latitude of deployment" ;
+		DEPLOYMENT_LATITUDE:standard_name = "latitude" ;
+		DEPLOYMENT_LATITUDE:units = "degrees_north" ;
+		DEPLOYMENT_LATITUDE:valid_max = "90" ;
+		DEPLOYMENT_LATITUDE:valid_min = "-90" ;
+	double DEPLOYMENT_LONGITUDE ;
+		DEPLOYMENT_LONGITUDE:long_name = "longitude of deployment" ;
+		DEPLOYMENT_LONGITUDE:standard_name = "longitude" ;
+		DEPLOYMENT_LONGITUDE:units = "degrees_east" ;
+		DEPLOYMENT_LONGITUDE:valid_max = "180" ;
+		DEPLOYMENT_LONGITUDE:valid_min = "-180" ;
+	string TELECOM_TYPE ;
+		TELECOM_TYPE:long_name = "type of telecommunication systems used by the glider" ;
+		TELECOM_TYPE:telecom_type_vocabulary = "TBD" ;
+	string TRACKING_SYSTEM ;
+		TRACKING_SYSTEM:long_name = "type of tracking systems used by the glider" ;
+		TRACKING_SYSTEM:tracking_system_vocabulary = "TBD" ;
+	string PLATFORM_TYPE ;
+		PLATFORM_TYPE:long_name = "type of glider" ;
+		PLATFORM_TYPE:platform_type_vocabulary = "https://vocab.nerc.ac.uk/collection/L06/current/";
+	string PLATFORM_MODEL ;
+		PLATFORM_MODEL:long_name = "model of the glider" ;
+		PLATFORM_MODEL:platform_model_vocabulary = "http://vocab.nerc.ac.uk/collection/B76/current/B7600027/";
+	string PLATFORM_MAKER ;
+		PLATFORM_MAKER:long_name = "Glider manufacturer" ;
+		PLATFORM_MAKER:platform_maker_vocabulary = "TBD" ;
+	string PLATFORM_SERIAL_NUMBER ;
+		PLATFORM_SERIAL_NUMBER:long_name = "glider serial number" ;
+	string PLATFORM_CODE ;
+		PLATFORM_CODE:long_name = "nickname of the glider" ;
+	int PLATFORM_DEPTH_RATING ;
+		PLATFORM_DEPTH_RATING:long_name = "Depth limit in meters of the glider for this mission" ;
+	string WMO_IDENTIFIER ;
+		WMO_IDENTIFIER:long_name = "wmo id" ;
+        int INSTRUMENT_CTD ;
```

Comment:
Is this effectively an empty variable or are you expecting an int identifier for the CTD?